Assets/Scripts/UI/Sun/AnimationSpeedInputField.cs

```diff
@@ -6,44 +7,52 @@ namespace Netherlands3D.Twin
 {
     public class AnimationSpeedInputField : MonoBehaviour
     {
+        [SerializeField] private int matchingDropdownIndex;
```

Comment:
Klein ding: maar de naamgeving is voor mij niet duidelijk; zelfs met het lezen van de code snap ik niet wat hier ingesteld wordt en wat de functie is

Reply:
dit is welke dropdown waarde bij deze field hoort, omdat te twee fields allebei ander gedrag hebben (de een moet de tijd weergeven, de andere een symbool) zijn er 2 gameObjects en moet alleen als de dropdown naar de juiste waarde verandert er iets gebeuren. ik zal nog even kijken of ik dit anders kan doen